cli/src/main.rs

```diff
@@ -0,0 +1,65 @@
+use std::process::{Command, Stdio};
+
+use clap::{command, Parser, Subcommand};
+pub mod new_project;
+use new_project::*;
+
+#[derive(Subcommand, Debug)]
+enum CliCommand {
+    #[command(about = "Create new Solana program")]
+    New(NewArgs),
+
+    #[command(about = "Compile/build program")]
+    Build,
+
+    #[command(about = "Execute all tests")]
+    Test(TestArgs),
+}
+
+#[derive(Parser)]
+struct Cli {
+    #[command(subcommand)]
+    command: CliCommand,
+}
+
+#[derive(Parser, Debug)]
+pub struct TestArgs {
+    /// Show test output (don’t capture stdout/stderr)
+    #[arg(long)]
+    pub nocapture: bool,
+}
+
+pub fn test_project(args: TestArgs) -> anyhow::Result<()> {
+    let mut command = Command::new("cargo");
+    command.arg("test-sbf");
+
+    if args.nocapture {
+        command.arg("--").arg("--nocapture");
+    }
+
+    command
+        .stdout(Stdio::inherit())
+        .stderr(Stdio::inherit())
+        .status()
+        .expect("Failed to execute command");
+    Ok(())
+}
```

Comment:
`star_frame` uses standard rust testing mostly, test-sbf is uncommon in solana development now too. Probably just remove this for now, we have plans for a more comprehensive test system.